scripts/assign_workflow_seqlab_quantstudio.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python
+from egcg_core.config import cfg
+
 from EPPs.common import StepEPP, get_workflow_stage, find_newest_artifact_originating_from
 
 
 class AssignWorkflowSeqLabQuantStudio(StepEPP):
-    _use_load_config = False  # prevent the loading of the config file
+    _use_load_config = True  # prevent the loading of the config file
```

Comment:
We're not preventing the loading of the config file if `_use_load_config` is now `True`

Reply:
Removed line as _use_load_config defaults as True